### PR TITLE
[7.0] Add purchase_supplier_postage_free

### PR DESCRIPTION
**Pull request by guewen**
- Add a field "Postage free above amount" on the suppliers, which is an
  optional amount above which the supplier does not charge the shipping
  fees.
- Display this amount on the purchase orders as an indication
- Add a filter on the purchase orders showing the ones which have reached
  the postage free amount.

### CODE REVIEW

File: purchase_supplier_postage_free/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{'name': 'Purchase Supplier Postage Free',
+ 'version': '1.0',
+ 'author': 'Camptocamp',
```
Comment (gurneyalex) [inline purchase_supplier_postage_free/__openerp__.py]: please add "Odoo Community Association (OCA)"

**Comment by lepistone** [comment]
I'm restarting the build

**Comment by lepistone** [comment]
I deleted a commit that added a dependency on `OCA/web`, that should not be needed. Let's see how the build goes. Otherwise :+1:

**Comment by guewen** [comment]
I am about to port this module to 8.0

**Comment by guewen** [comment]
> I deleted a commit that added a dependency on OCA/web, that should not be needed. Let's see how the build goes. Otherwise :+1:

The dependency on `web` is needed for `web_context_tunnel`.

**Comment by pedrobaeza** [comment]
I think this is better treated using delivery method in purchases, that already includes free postage. An approximation has been already done here for v8: https://github.com/OCA/purchase-workflow/pull/157

**Comment by gurneyalex** [comment]
runbot should be green once #163 is merged and this PR is rebased.

**Comment by jgrandguillaume** [comment]
I'm :+1:  for the reasons given here: https://github.com/OCA/purchase-workflow/pull/159#issuecomment-131098621

**Comment by guewen** [comment]
Rebased

File: purchase_supplier_postage_free/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ 'version': '1.0',
+ 'author': 'Camptocamp,Odoo Community Association (OCA)',
+ 'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
+ 'category': 'Purchase Management',
+ 'depends': ['purchase',
+             'web_context_tunnel',
+             ],
+ 'description': """
+Purchase Supplier Postage Free
```
Comment (pedrobaeza) [inline purchase_supplier_postage_free/__openerp__.py:32]: Can you please follow full OCA's description template?

**Comment by pedrobaeza** [comment]
@jgrandguillaume, @guewen. OK, I think we can have both modules co-existing for the reasons you gave.

File: purchase_supplier_postage_free/__openerp__.py (hunk 2 above)
Comment (guewen) [inline purchase_supplier_postage_free/__openerp__.py:32]: Done

**Comment by pedrobaeza** [comment]
I mention in 8.0 PR about renaming the module. What do you think?

**Comment by pedrobaeza** [comment]
As this is very old and there's another PR for version 8, I'm going to close this PR, but if you still think this is useful, please reopen, but fixing Travis status.

**Pull request closed without merging**